Assets/MixedRealityToolkit.SDK/Features/UX/Prefabs/Menus/NearMenu3x1.prefab

```diff
@@ -10,7 +10,7 @@ PrefabInstance:
     - target: {fileID: 3515418990045158255, guid: 40b588bd40632eb41a0af374d91c4fc9,
         type: 3}
       propertyPath: m_Name
-      value: MenuExample3x1
+      value: NearMenu3x1
```

Comment:
Are these renamed breaking changes? If they are we need to mention in release notes, and consider not doing them unless there's a good reason.

Reply:
I moved these prefabs from the Examples folder to SDK and renamed them. It does not break the existing example scene (NearInteractionExamples.unity) since GUIDs are not changed.

Reply:
We should still update the 2.4 release notes to make mention of the rename. In case, people were switching NearMenuExample3x1....I mean the names are close enough so people should find it.

Only other potential issue here is if somehow downloaded the unitypackage for examples and not the SDK?